climesync.py

```diff
@@ -220,17 +327,39 @@ def disconnect():
     return list()
 
 
-def sign_in():
+def sign_in(arg_user="", arg_pass="", config_dict=dict()):
     """Attempts to sign in with user-supplied or command line credentials"""
 
-    global arg_username, arg_password, ts
+    global ts
 
     if not ts:
         return {"error": "Not connected to TimeSync server"}
 
-    # If username or password not provided on command line, ask for them
-    username = arg_username if arg_username else raw_input("Username: ")
-    password = arg_password if arg_password else raw_input("Password: ")
+    username = ""
+    password = ""
+
+    # If username or password in config, use them at program startup.
+    if arg_user:
+        username = arg_user
+
```

Comment:
Please remove the spacing between `if/elif/else` blocks so the blocks are easier to parse as being contiguous.
```
if stuff
    foo

else
    baz

if  morestuff
    bar

elif morestuff
    foobee
```
is harder to parse than
```
if stuff
    foo
else
    baz

if  morestuff
    bar
elif morestuff
    foobee
```